src/NServiceBus.Core/Recoverability/SecondLevelRetries/Config/FirstLevelRetriesSettings.cs

```diff
@@ -0,0 +1,27 @@
+namespace NServiceBus.SecondLevelRetries.Config
```

Comment:
This namespace seems wrong. Should it either be in `NServiceBus` or some other sub-namespace?